src/locales/en.json5

```diff
@@ -321,8 +321,7 @@
     "session": "Session",
     "submission": "Submission",
     "submissions": "Submissions",
-    "projectRoles": "Project Roles",
-    "formPreview": "Form Preview"
```

Comment:
Isn't this message still used for the title of the form preview page? I think I see it being used in src/routes.js.